sites/brussel/macros.tex

```diff
@@ -27,6 +27,6 @@
 \macro{\computenode}{r5e3cn13.hydra.brussel.vsc}
 \macro{\computenodeshort}{r4e2cn03}
```

Comment:
change these too

Reply:
but then the pictures have to be changed as well. I am not ready to do that yet.